composer.json

```diff
@@ -37,9 +37,6 @@
             "stubs"
         ]
     },
-    "conflict": {
-        "rector/rector": "<0.11"
```

Comment:
Alternative to `require` would be adding here a conflict to `>=0.13`.

@TomasVotruba what do you recommend?

Reply:
This is not needed, as the require section takes care of it.